include/rocksdb/options.h

```diff
@@ -952,16 +952,19 @@ struct DBOptions {
   AccessHint access_hint_on_compaction_start = NORMAL;
 
   // The size RocksDB uses to perform readahead during compaction read.
-  // If set zero, RocksDB will sanitize it to be 2MB during db open.
   // If you're
   // running RocksDB on spinning disks, you should set this to at least 2MB.
   // That way RocksDB's compaction is doing sequential instead of random reads.
   //
   //
-  // Default: 0
+  // Default: 2MB
+  //
+  // For direct IO, if set 0, RocksDB will sanitize the value to the default.
+  // For non-direct IO, if set 0, RocksDB will disable compaction readahead
+  // instead.
```

Comment:
Do we need this distinction? How about never sanitize zero?

Reply:
Not needed as the meta approach changes